osf/migrations/0142_change_registration_schemas.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import unicode_literals
+
+import logging
+
+from django.db import migrations
+from osf.utils.migrations import ensure_schemas, remove_schemas
+
+
+logger = logging.getLogger(__file__)
+
+def update_schemas(state, schema):
+    remove_schemas()
```

Comment:
Is it necessary to remove the schemas before adding them?

We have a number of migrations that run `ensure_schemas` and none of them call `remove_schemas` first.

Reply:
Those migrations add schemas, when schemas are modified they aren't saved by ensure schemas, at least that's my experience in the shell.

Reply:
Looks like ensure_schemas calls `update_or_create`, which should update the schema.

I think that removing the schemas will break all relationships between registrations and their metaschema.